scripts/run.sh

```diff
@@ -2,4 +2,4 @@
 
 killall sifnoded
 
-sifnoded start --trace
+sifnoded start --trace --home ~/.sifnoded/ --p2p.laddr 0.0.0.0:27655  --grpc.address 0.0.0.0:9096 --grpc-web.address 0.0.0.0:9093 --address tcp://0.0.0.0:27659 --rpc.laddr tcp://127.0.0.1:26657
```

Comment:
@Jedi2002 should we use 
```
--rpc.laddr tcp://127.0.0.1:26657
```

or 

```
--rpc.laddr tcp://0.0.0.0:26657
```

considering the other params using 0.0.0.0

Reply:
We can use anything tbh, I assume this is just a test script? Not used for any actual deployment